addon/templates/components/frost-bunsen-input-date.hbs

```diff
@@ -0,0 +1,22 @@
+<label class={{labelWrapperClassName}}>
+  {{renderLabel}}
+  {{#if required}}
+    <div class='frost-bunsen-required'>Required</div>
+  {{/if}}
+</label>
+<div class={{inputWrapperClassName}}>
+  {{frost-date-picker
+    class=valueClassName
+    currentValue=(readonly transformedValue)
+    disabled=disabled
+    hook=(concat hook '-datePicker')
+    onFocusIn=(action 'hideErrorMessage')
+    onFocusOut=(action 'showErrorMessage')
+    onSelect=(action 'selectDate')
+  }}
+</div>
+{{#if renderErrorMessage}}
+  <div class="frost-bunsen-error">
+    {{renderErrorMessage}}
+  </div>
+{{/if}}
```

Comment:
I know it is minor but can you add a newline to the end of the file (you can probably configure your editor to always do this for you on save).